### DISCUSSION TITLE
Where I can the endpoint when forwarding to webhook?

### DISCUSSION THREAD

**Discussion by GilbertMizrahi** [forwardemail/forwardemail.net#230] [Q&A] [answered]
I'm trying to setup an alias to forward emails to a webhook.
I followed the instructions, but I can't find the endpoint I should use to get the response from the webhook.

I will also appreciate some info on how to use that webhook in an application.

**Comment by titanism** [comment, marked as answer]
https://forwardemail.net/en/faq#do-you-support-webhooks
